test/string_test.rb

```diff
@@ -85,6 +87,20 @@
     end
   end
 
+  describe '#classified' do
```

Comment:
@jodosha I added the lines 73 & 74 to test the original issue. Should I make a new test case for that?